src/DocRaptor.php

```diff
@@ -71,10 +67,19 @@ public function setDocumentUrl($document_url)
     /**
      * @param string $document_type
      * @return $this
+     * @throws Exception
      */
     public function setDocumentType($document_type)
     {
+        $allowedValues = array('xls', 'xlsx', 'pdf');
+
+        if (! in_array(strtolower(trim($document_type)), $allowedValues)) {
+            throw new Exception('Value not accepted by method.');
+        }
+
         $document_type = strtolower($document_type);
+
+        //@todo why is xls cast to pdf?
         $this->type = $document_type == 'pdf' || $document_type == 'xls' ? $document_type : 'pdf';
```

Comment:
This was a null check, basically. You can remove this now given you are doing the check above?